### PR TITLE
fix keyboard controls lost issue

### PR DESCRIPTION
**Pull request by Algorush**

### CODE REVIEW

**Comment by kfarr** [comment]
it does not work, it has odd behavior
video:

https://user-images.githubusercontent.com/470477/235339029-c10fa746-a859-4839-8fe4-e5dbc01492f5.mov

File: src/lib/raycaster.js [hunk 1]
```diff
@@ -127,9 +127,15 @@ export function initRaycaster(inspector) {
...
       mouseCursor.setAttribute('raycaster', 'enabled', true);
+      inspector.container.addEventListener('mousedown', onMouseDown);
+      inspector.container.addEventListener('mouseup', onMouseUp);
+      inspector.container.addEventListener('dblclick', onDoubleClick);
     },
     disable: () => {
       mouseCursor.setAttribute('raycaster', 'enabled', false);
+      inspector.container.removeEventListener('mousedown', onMouseDown);
+      inspector.container.removeEventListener('mouseup', onMouseUp);
+      inspector.container.removeEventListener('dblclick', onDoubleClick);
```
Comment (vincentfretin) [inline src/lib/raycaster.js]: As a reminder, you have the dblclick listener commented on your project, so you don't want to add it back here, comment it here as well.

Comment (Algorush) [reply src/lib/raycaster.js]: @vincentfretin Thanks! I didn't pay attention to that.

**Comment by Algorush** [comment]
> 

@kfarr , sorry, I couldn't check on my side, after updating npm I couldn't deal with package conflicts yesterday... To make a new dist

**Comment by Algorush** [comment]
so far the simplest solution I see is to call `document.activeElement.blur()` after closing the 'Editor'. Maybe not the best solution. Then the keyboard works immediately after closing the Editor. Better of course to find the reason why the focus stays on the button, but I haven't found it yet
https://github.com/3DStreet/3dstreet-editor/pull/161

**Comment by kfarr** [comment]
@Algorush I'm ok with using the `blur` workaround you suggest; another idea instead of removing focus (blur) of active element, would an alternative be to proactively focus on the a-frame scene canvas? Is that the desired state?

**Comment by kfarr** [comment]
merged this one instead: https://github.com/3DStreet/3dstreet-editor/pull/161

**Pull request merged**